packages/components/package.json

```diff
@@ -30,10 +30,12 @@
 	],
 	"dependencies": {
 		"@babel/runtime": "^7.13.10",
-		"@emotion/cache": "^10.0.27",
-		"@emotion/core": "^10.1.1",
-		"@emotion/css": "^10.0.22",
-		"@emotion/styled": "^10.0.23",
+		"@emotion/cache": "^11.1.3",
+		"@emotion/core": "^11.0.0",
+		"@emotion/css": "^11.1.3",
+		"@emotion/native": "^11.0.0",
```

Comment:
`"@emotion/native"` is RN dependency and it should not be listed here because it pulls in additional dependencies that cause conflicts after publishing to npm. In general, we don't list RN dependencies for individual packages. It can stay listed as a dependency in the root `package.json` file.